src/main/java/hudson/remoting/Launcher.java

```diff
@@ -189,6 +189,10 @@ public boolean verify(String s, SSLSession sslSession) {
     @Option(name="-noReconnect",usage="Doesn't try to reconnect when a communication fail, and exit instead")
     public boolean noReconnect = false;
 
+    @Option(name = "-noKeepAlive",
+            usage = "Do not open the socket to the master with SO_KEEPALIVE enabled")
```

Comment:
@olivergondza My plan is to create a documentation page, which will clarify options and their potential impact. By now I think the current summary string is acceptable. Do you agree?